tiktok_reporter_analysis/extract_frames.py

```diff
@@ -2,12 +2,14 @@
 import os
 import shutil
 
-import cv2
+from moviepy.editor import VideoFileClip
 
 logger = logging.getLogger(__name__)
 
 
 def extract_frames_from_video(video_path, output_folder):
+    from tiktok_reporter_analysis.common import extract_frames
```

Comment:
A minor detail but this could be imported outside the function

Reply:
Fixed :)